_layouts/project.html

```diff
@@ -62,13 +62,13 @@ <h5 class=''>
         <li class='thumbnail'>
           <a href='{{ next_project.url }}'>
             {% include img.html image=next_project.thumb_image %}
-            <h2 class="h4">{{ next_project.title }}</h2>
+            <h2 class="thumbnail-title">{{ next_project.title }}</h2>
           </a>
         </li>
         <li class='thumbnail'>
           <a href='{{ nextNext_project.url }}'>
             {% include img.html image=nextNext_project.thumb_image %}
-            <h2 class="h4">{{ nextNext_project.title }}</h2>
+            <h2 class="thumbnail-title">{{ nextNext_project.title }}</h2>
```

Comment:
Sorry one more change...and this is mostly my fault. The thumbnails are used in other areas of the site. It's not a very DRY use right now :( You would either need to refactor thing so we have a thumbnail include (the cleanest way) and then replace everywhere that a thumbnail is used or add the `.thumbnail-title` class to everywhere a thumbnail is used (the faster way). Right now I can see that it's used in `_includes/thumbnails.html`